src/ocean/task/IScheduler.d

```diff
@@ -307,12 +308,10 @@ public interface IScheduler
 *******************************************************************************/
 
 public IScheduler theScheduler ( )
-in
-{
-    assert(_scheduler !is null, "Scheduler is null, initScheduler must be called before using it");
-}
-body
 {
+    verify(_scheduler !is null,
+        "Scheduler is null, initScheduler must be called before using it");
+
```

Comment:
I think this one stay as is, though. Or change this `verify` to `assert`. This is the condition that can fail only once per runtime.

Reply:
you mean remove the `in` contract and keep the `assert()`, right?

Reply:
Yeah :+1: